spec/lib/geo_combine/iso19139_spec.rb

```diff
@@ -33,4 +33,9 @@
       expect(iso_object.to_html).to be_an String
     end
   end
+  describe '#to_geoblacklight#solr_geom' do
```

Comment:
Have you considered putting this within the `to_geoblacklight` describe block? Then you could use the already created `valid_geoblacklight` object.